package.json

```diff
@@ -1,13 +1,33 @@
 {
+    "name": "initial-twitter-offering",
+    "version": "1.0.0",
+    "license": "MIT",
+    "author": "Yisi Liu",
+    "main": "hardhat.config.ts",
+    "scripts": {
+        "test": "hardhat test",
```

Comment:
suggest, integrate husky on pre-commit, automate compile and test

Reply:
I'd like to have a try